### PR TITLE
Expanding Lobster's Cyrillic

### PR DESCRIPTION
**Pull request by alexeiva**
@ThomasJockin

Hey Thomas,

I am expanding Lobster's Cyrillic. Please don't merge this request yet, until all the work in complete. 

Cheers, 
-a

### CODE REVIEW

**Comment by alexeiva** [comment]
@m4rc1e is this ready for merging now?

**Comment by m4rc1e** [comment]
@alexeiva I need to do a once over myself. 

I'll send him a pr and close this one.

Cheers,
Marc

**Comment by alexeiva** [comment]
@m4rc1e good. thank you

**Comment by m4rc1e** [comment]
@alexeiva wouldn't it be better to pr this to @impallari's repo?

**Comment by m4rc1e** [comment]
Alright, I've just done a quick review of this pr

---

screen shot 2017-10-23 at 11 38 21
screen shot 2017-10-23 at 11 37 30

Are these the correct shape?

They usually have a base which looks like this

screen shot 2017-10-23 at 11 40 10


---

These are missing 

screen shot 2017-10-23 at 11 36 34

---

The quote right has gotten smaller

lobster-quote-single

**Comment by alexeiva** [comment]
> @alexeiva wouldn't it be better to pr this to @impallari's repo?

I think this PR is a duplicate of https://github.com/impallari/The-Lobster-Font/pull/5. It's fine to close it.


Thanks for the review, I will get this fixed in the current expansion round.

`ғ` U+0493 can have a handwritten form too as in the case. One of the early adopters of this approach is Lucas' Calibri font.

`cyrbreve` and `cyrBreve` are obsolete (Adobe PUA U+F6D6 and U+F6D4). At the moment we use unencoded `brevecomb-cy` and `brevecomb-cy.case` for this.

To sum up, here is todo :

— add `commaaccent`
— fix `quoteright`

**Comment by alexeiva** [comment]
Here is a historical sample of base glyph `г` and  `ӷ` `U+04F7` handrwitten

screen shot 2017-10-23 at 4 42 41 pm

**Comment by m4rc1e** [comment]
@ThomasJockin could you merge this please?
